group_vars/all/main.yml

```diff
@@ -114,3 +114,34 @@ openshift_volumes:
   - openshift/edxapp/pvc/static.yml
   - openshift/richie/pvc/media.yml
   - openshift/richie/pvc/static.yml
+
+  # default users for development environment
+demo_hashed_password: 'pbkdf2_sha256$20000$TjE34FJjc3vv$0B7GUmH8RwrOc/BvMoxjb5j8EgnWTt3sxorDANeF7Qw=' # edx
+demo_test_users:
+  - email: 'honor@example.com'
+    username: honor
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: false
+    is_superuser: false
+  - email: 'audit@example.com'
+    username: audit
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: false
+    is_superuser: false
+  - email: 'verified@example.com'
+    username: verified
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: false
+    is_superuser: false
+demo_staff_users:
+  - email: 'staff@example.com'
+    username: staff
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: true
+    is_superuser: false
+  - email: 'admin@example.com'
+    username: admin
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: true
+    is_superuser: true
+demo_test_staff_users: "{{ demo_test_users }} + {{ demo_staff_users }}"
```

Comment:
A better name may be `demo_all_users` ??

Reply:
I was reusing edX variable names